src/managers/GuildEmojiManager.js

```diff
@@ -119,6 +121,10 @@ class GuildEmojiManager extends BaseManager {
     if (emojiResolvable) return emojiResolvable.identifier;
     if (emoji instanceof ReactionEmoji) return emoji.identifier;
     if (typeof emoji === 'string') {
+      const res = parseEmoji(emoji);
+      if (res.name.length) {
```

Comment:
It might be null:
https://github.com/discordjs/discord.js/blob/77b6a7d5bd7463f9f397179f5897ec0d00706749/src/util/Util.js#L262-L268
```suggestion
      if (res && res.name.length) {
```